apps/explorer/src/pages/BlockInfoPage.tsx

```diff
@@ -311,3 +287,9 @@ const IODetailList = styled.div`
   border-radius: 8px;
   padding: 15px;
 `
+
+const UncleBlocks = styled.div`
+  display: flex;
+  flex-direction: column;
+  gap: 5px;
+`
```

Comment:
Seems like a formatting error here

Reply:
Why? It's just the end line indicator I believe?